### PR TITLE
Rek movedate select

### PR DESCRIPTION
**Pull request by kilbergr**
## Description

Add the date picker to the HHG flow. Date turns blue on selection and static text appears.
I suspect there are a number of things that could be done better here. Also am uncertain how to keep react/redux state without any API work.

## Reviewer Notes

This assumes a fair amount of [another task](https://www.pivotaltracker.com/story/show/158776036) will be done a certain way.
The ducks file is in an anticipatory state because there's nothing in the database around HHGs (shipments) yet. The form is not fully functional, so there's no redux state saving happening. 

## Code Review Verification Steps

* [x] All tests pass.
* [x] There are no aXe warnings for UI.
* [x] This works in IE.
* [x] Request review from a member of a different team.
* [x] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/158859268) for this change

## Screenshots
hhgselect

### CODE REVIEW

File: src/scenes/Moves/Hhg/DatePicker.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+          <div className="usa-width-one-third">
+            <DayPicker
+              onDayClick={this.handleDayClick}
+              selectedDays={this.state.selectedDay}
+            />
+          </div>
+
+          <div className="usa-width-two-thirds">
+            {this.state.showInfo && (
+              <table>
```
Comment (tinyels) [inline src/scenes/Moves/Hhg/DatePicker.jsx]: You might want to use todo styling for this. I'm also not sure it's needed for coal scuttle.

File: src/scenes/Moves/Hhg/ducks.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+export function hhgReducer(state = initialState, action) {
+  switch (action.type) {
+    case GET_LOGGED_IN_USER.success:
+      // Initialize state when we get the logged in user
+      const activeOrders = fetchActive(
+        get(action.payload, 'service_member.orders'),
+      );
+      const activeMove = fetchActive(get(activeOrders, 'moves'));
+      const activeHhg = fetchActive(get(activeMove, 'shipments'));
+      return Object.assign({}, state, {
```
Comment (tinyels) [inline src/scenes/Moves/Hhg/ducks.js]: I'd like to talk to y'all about normalizing this.

File: src/scenes/Moves/Hhg/DatePicker.jsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      example: '2018-04-26',
+      title: 'Move Date',
+      'x-nullable': true,
+      'x-always-required': true,
+    },
+  },
+};
+const HhgDateWizardForm = reduxifyWizardForm(formName);
+
+export class HhgDatePicker extends Component {
```
Comment (jim) [inline src/scenes/Moves/Hhg/DatePicker.jsx]: Naming this class`HHGDatePicker` would align with the [naming conventions](https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms) mentioned in our Frontend guide.

File: src/scenes/Moves/Hhg/ducks.js (hunk 2 above)
Comment (jim) [reply src/scenes/Moves/Hhg/ducks.js]: We're going to have a convo next week with @tinyels about how to best implement this. That work will be part of the next set of HHG stories.

**Comment by jim** [comment]
@kilbergr Looks like `make client_build` is failing!

File: src/scenes/Landing/index.jsx [hunk 4]
```diff
@@ -131,6 +139,7 @@ export class Landing extends Component {
 }
 
 const mapStateToProps = state => ({
+  reduxState: state,
```
Comment (tinyels) [inline src/scenes/Landing/index.jsx]: what is this?

**Pull request merged**